common/app/views/fragments/pagination.scala.html

```diff
@@ -28,11 +28,7 @@
 
 @paginationExtreme(pageNo: Option[Int], dir: String) = {
```

Comment:
Does this function still need to accept the second param? Looks like it's always called with `"first"`

Reply:
Yeah you're correct, will remove the 2nd param and the `@if(dir == "first")` check...